src/rest.ts

```diff
@@ -1,10 +1,13 @@
 import cockpit from "cockpit";
 
-import { debug } from "./util.js";
+import { debug } from "./util.tsx";
```

Comment:
I think if we change anything, we should drop the suffix completely in import statements.